lib/fluent/plugin/out_kafka_buffered.rb

```diff
@@ -83,8 +83,20 @@ class Fluent::KafkaOutputBuffered < Fluent::BufferedOutput
 DESC
 
   config_param :monitoring_list, :array, :default => [],
-               :desc => "library to be used to monitor. statsd and datadog are supported"
-
+               :desc => "library to be used to monitor. statsd and datadog are supported",
+               :obsoleted => "Use <monitoring> section instead."
```

Comment:
Using `deprecated` is better for the migration first.
After several release, we can use `obsoleted`.